Description: Remove 'const' to prevent build failures with GCC4.6.
Origin: http://dvbcut.svn.sourceforge.net/viewvc/dvbcut/trunk/src/dvbcut.cpp?r1=173&r2=172&pathrev=173
Applied-Upstream: yes 
---
 src/dvbcut.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- dvbcut.orig/src/dvbcut.cpp
+++ dvbcut/src/dvbcut.cpp
@@ -1325,7 +1325,7 @@ void dvbcut::eventlistcontextmenu(QListB
   if (lbi->rtti()!=EventListItem::RTTI())
     return;
   // is it a problem to have no "const EventListItem &eli=..."? Needed for seteventtype()...! 
-  EventListItem &eli=*static_cast<const EventListItem*>(lbi);
+  EventListItem &eli=*static_cast<EventListItem*>(lbi);
 
   QPopupMenu popup(eventlist);
   popup.insertItem("Go to",1);
